src/main/resources/languages/VI.yml

```diff
@@ -0,0 +1,114 @@
+Layout_Set_Status: 'Chỉnh biễu tượng trạng thái cho bản thân'
+Layout_Set_Money: 'Chỉnh biểu tượng tiền cho bản thân'
+Layout_Set_Other_Money: 'Chỉnh biểu tượng tiền cho người khác'
+Help_TradeSystem: '&7Dùng &e/%label% <layout, reload>'
+Help_TradeSystem_Layout: '&7Dùng &e/%label% layout <create, edit, delete, activate>'
+Help_TradeSystem_Layout_Activate: '&Dùng &e/%label% layout activate <name>'
+Help_TradeSystem_Layout_Delete: '&7Dùng &e/%label% layout delete <name>'
+Help_TradeSystem_Layout_Edit: '&7Dùng &e/%label% layout edit <name>'
+Layout_Does_Not_Exist: '&cKhung này không tồn tại!'
+Layout_Activated: '&7Khung ''&b%name%&7'' đã được &akích hoạt&7.'
+Layout_Deleted: '&7Khung ''&b%name%&7'' đã bị &cxóa&7.'
+Enter_Correct_Name_Space: '&cKhông dùng dấu cách.'
+Sure_That_You_Want_To_Loose_Your_Data: '&7Bạn có chắc bạn &ckhông &7muốn &clưu &7thay đổi của bạn?'
+Close: 'Đóng'
+Cannot_Delete_Standard: '&7Bạn không thể &cxóa &7khung &ctiêu chuẩn&7!'
+Cannot_Edit_Standard: '&7Bạn không thể &cchỉnh &7khung &ctiêu chuẩn&7!'
+Layout_Already_Activated: '&7Khung này &cđã được kích hoạt&7!'
+Layout_Set_Replacement_For_Money: '&7Biểu tượng &ethay thế biểu tượng tiền&7, khi &ekhông có plugin tiền &7được tìm thấy.'
+Layout_Hint_Same_Slot_Amount: '&7Cả 2 người giao dịch &cphải có số ô như nhau&7!'
+Current_Amount: 'Số lượng hiện tại'
+Trade_Spam: '&7Người chơi này đã &cnhận yêu cầu giao dịch rồi&7!'
+Plugin_Reloading: '&cPlugin sẽ được nạp lại...'
+Success_Plugin_Reloaded: '&aPlugin đã được nạp lại.'
+Player_is_not_in_range: '&7Người chơi &c%player% &7đang &ckhông ở gần&7!'
+Cannot_trade_in_that_GameMode: '&7Bạn &ckhông thể giao dịch &7ở chế độ này!'
+Other_cannot_trade_in_that_GameMode: '&7Người chơi &ckhông thể giao dịch &7ở chế độ này!'
+Trade_cancelled_by_attack: '&cGiao dịch đã hủy do có sự tấn công.'
+Trade_Partner_No_Space: '&7Bạn giao dịch của bạn &ckhông còn chỗ trống &7trong kho của họ!'
+Trade_Offline: '&7Bạn &ckhông còn nhận &7yêu cầu giao dịch nữa!'
+Trade_Online: '&7Bạn bây giờ &acó thể nhận &7yêu cầu giao dịch!'
+Trade_Partner_is_Offline: '&7Người chơi &ckhông còn nhận &7yêu cầu giao dịch!'
+Trade_You_are_Offline: '&7Dùng "%command%&e/trade toggle%command%&7" để giao dịch lại.'
+Trade_Placed_Blocked_Item: '&7Vât phẩm này &ckhông thể &7giao dịch!'
+Cannot_trade_in_bed: '&7Bạn &ckhông thể giao dịch&7, khi đang ở trên giường.'
+Cannot_trade_in_world: '&7Bạn &ckhông thể giao dịch &7ở thế giới này!'
+Other_cannot_trade_in_world: '&7Người chơi §ckhông thể giao dịch &7ở thế giới của họ!'
+Cannot_trade_while_invisible: '&7Bạn &ckhông thể giao dịch &7khi đang &ctàng hình&7!'
+Other_is_already_trading: '&7Người chơi này hiện tại &cđang giao dịch&7!'
+Online: Trực tuyến
+Offline: Ngoại tuyến
+Your_request_epired: '&7Yêu cầu giao dịch đến &e%player% &7đã &chết hạn&7.'
+Request_expired: '&7Yêu cầu giao dịch từ &e%player% &7đã &chết hạn&7.'
+Fancy_Countdown: ' &7- &e%seconds% giây'
+Economy_Error: '&7Giao dịch &cđã hủy &7vì &clỗi thị trường&7.'

```

Comment:
Lines 116 & 117 missing.
* l. 116: `Items_Dropped: '&7Your inventory was &cfull&7, some items were &cdropped on the ground&7!'`
* l. 117: `TradeLog_Disabled: "&7Trade logging is &cdisabled&7!"`